en/key_concepts.rst

```diff
@@ -18,67 +21,98 @@ There are two types of nodes in Open Orchestra:
 
 See also the documentation page about `node parameters`_.
 
-Content types
--------------
+Zones
+~~~~~
 
-Content types are a way to create new kinds of content that will be available to contribution.
-Creating a content type allows to define the fields available in the future contents.
+Zones are the way to organize the architecture of the page.
+They are embedded in the nodes and can contain other zones or blocks exclusively.
 
-For instance, a 'news' content will have differents fields from an 'article' content,
-so there would be two distinct content types for these objects : one for the news contents and another for the article contents.
+See also `configuring a zone`_.
 
-See also `Presentation of the Content Types`_.
+Blocks
+~~~~~~
+
+A block is the base brick representing any visible element on a page.
+Aggregating blocks found in the nodes will lead to the construction of the pages.
+
+See also the `list of blocks`_ available.
+
+Transverses blocks
+~~~~~~~~~~~~~~~~~~
+
+Transverses blocks are defined inside the transverse node, they can be shared by several nodes of a website.
+All blocks can be transverse, as long as they are added to the transverse node.
+Transverse blocks can only be conigured in the transverse node.
+
+Templates
+~~~~~~~~~
+
+Templates are used to preconfigure nodes by setting zones that are commonly used accross pages.
+Open Orchestra let the user start with a template when creating a node.
```

Comment:
I think this explanation is too vague. It could be clearer with a sentence like "Templates are patterns that can be used and reused to create nodes."